README.md

```diff
@@ -1,70 +1,45 @@
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `yarn eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
-
-### Analyzing the Bundle Size
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
-
-### Making a Progressive Web App
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
-
-### Advanced Configuration
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `yarn build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+University: San Jose State University
+
+Course: CMPE 172: Enterprise Software
+
+Professor: Sanjay Garje
+
+Students:
+Minh Phan
+Evan Ugarte
+Yanyan Yang
+Roy Zhang
+
+Project Introduction:
+- Upload, Delete and Search for Photos
+- Uses AWS Rekognition to automatically categorize photos when they are uploaded
+- Users can retrieve photos based on autogenerated tags
+
+Sample Demo Screenshots: TBD
+
+Pre-Requisites Setup:
+
+Cloud Services:
+- S3 buckets to hold images and stage website
+- DynamoDB tables to store user information and references to photo objects
+- Route53 for domain name and routing
+- ELB, ASG, and EC2 to host our server and direct traffic
+- CloudFront for content delivery
+- SNS and CloudWatch for notifications about our system
+- Rekognition for label detection on images
+- Cognito user pool for user authentication
+
+Local Software:
+- Visual Studio Code. ESlint plugin.
```

Comment:
I believe the markdown editor allows you to upload photos by copy+paste or with a link. see [here](https://guides.github.com/features/mastering-markdown/#examples)